haproxy_daemon.go

```diff
@@ -0,0 +1,223 @@
+// Copyright © 2018 Tuenti Technologies S.L.
```

Comment:
The code of this file is mostly the code removed from haproxy.go, the only meaningful change is the -D flag added to haproxy execution.